components/server/src/ome/services/ThumbnailBean.java

```diff
@@ -1005,7 +1005,15 @@ public void createThumbnailsByLongestSideSet(Integer size,
         // process due to the possible unloaded Pixels. If we do not,
         // Pixels will be unloaded and we will hit
         // IllegalStateException's when checking update events.
-        iUpdate.saveArray(toSave.toArray(new Thumbnail[toSave.size()]));
+        try {
+        	iUpdate.saveArray(toSave.toArray(new Thumbnail[toSave.size()]));
+        } catch (Exception e) {
```

Comment:
can this catch be more specific? also, maybe comment what it means if it happens?